tests/core/pyspec/eth2spec/test/altair/block_processing/test_process_sync_aggregate.py

```diff
@@ -206,7 +210,6 @@ def test_sync_committee_rewards_duplicate_committee_no_participation(spec, state
     committee_indices = get_committee_indices(spec, state, duplicates=True)
     committee_size = len(committee_indices)
     committee_bits = [False] * committee_size
-    assert len(committee_bits) == committee_size
```

Comment:
this likely got copied from another test where it was more important to check the committee size; however, there is no manipulation of this set here so we just remove the assert